tests/raftstore/test_conf_change.rs

```diff
@@ -416,8 +416,19 @@ fn test_split_brain<T: Simulator>(cluster: &mut Cluster<T>) {
     pd_client.must_remove_peer(r1, new_peer(2, 2));
     pd_client.must_remove_peer(r1, new_peer(3, 3));
 
-    // refresh region info, maybe no need
     cluster.must_put(b"k2", b"v2");
+    must_get_equal(&cluster.get_engine(6), b"k2", b"v2");
+    let region_detail = cluster.region_detail(r1, 1);
+    for peer in region_detail.get_region().get_peers() {
+        match peer.get_id() {
```

Comment:
In this case, the match can be written as `assert!(peer.get_id() < 4 || peer.get_id() > 6)`.